lavamoat/build-system/policy.json

```diff
@@ -1136,6 +1136,21 @@
         "string.prototype.matchall>side-channel": true
```

Comment:
pretty sure we don't want to commit these changes? I believe these are OS specific.

Reply:
i believe this comes from the auto generated lavamoat policies